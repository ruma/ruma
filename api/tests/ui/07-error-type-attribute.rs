use ruma_api::ruma_api;

ruma_api! {
    metadata: {
        description: "Does something.",
        method: POST, // An `http::Method` constant. No imports required.
        name: "some_endpoint",
        path: "/_matrix/some/endpoint/:baz",
        rate_limited: false,
        authentication: None,
    }

    request: {}

    response: {}

    #[derive(Default)]
    error: ruma_api::error::Void
}

fn main() {}
