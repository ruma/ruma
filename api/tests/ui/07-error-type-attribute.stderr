error: unexpected token
  --> $DIR/07-error-type-attribute.rs:17:5
   |
17 |     #[derive(Default)]
   |     ^
